### PR TITLE
[XB] Workaround to fix memory issue for Xbox One X

### PR DESCRIPTION
**Pull request by iuriionishchenko**
Number of the video decoder output buffers was decreased from 10 to 5

b/268591344

### CODE REVIEW

File: starboard/shared/win32/video_decoder.cc [hunk 1]
```diff
@@ -191,6 +193,15 @@ VideoDecoder::VideoDecoder(
       is_hdr_supported_(is_hdr_supported) {
   SB_DCHECK(output_mode == kSbPlayerOutputModeDecodeToTexture);
 
+  const int kMaxNameSize = 256;
+  char value[kMaxNameSize];
+  bool result = SbSystemGetProperty(kSbSystemPropertyModelName, value, kMaxNameSize);
```
Comment (jasonzhangxx) [inline starboard/shared/win32/video_decoder.cc:198]: Can we reuse starboard::shared::uwp::GetXboxType()?

Comment (iuriionishchenko) [reply starboard/shared/win32/video_decoder.cc]: Done

**Comment by kaidokert** [comment]
> Thanks for your pull request! It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

Please address this.

**Review by amurovanyi — APPROVED** [review]

**Comment by kaidokert** [comment]
Merge conflict needs to be resolved, and please try to get the checks green

File: starboard/shared/win32/video_decoder.cc (hunk 1 above)
Comment (iuriionishchenko) [reply starboard/shared/win32/video_decoder.cc]: Using ::starboard::shared::uwp::GetXboxType() causes build issue for win32 project. Should I add refences to uwp name space  in the win32 project or it is bate to move ::starboard::shared::uwp::GetXboxType() to some common namespace?

Comment (amurovanyi) [reply starboard/shared/win32/video_decoder.cc]: We've verified that proposed reduction of the queue length doesn't affect Cobalt performance in both SDR and HDR modes. The number of frame drops is not changed for 30 and 60 fps videos.

**Review by jasonzhangxx — APPROVED** [review]

**Comment by kaidokert** [comment]
Hm, windows builds still failing with what look like a somewhat related error, missing dependency ? @TyHolc perhaps have a look.

**Comment by TyHolc** [comment]
> Hm, windows builds still failing with what look like a somewhat related error, missing dependency ? @TyHolc perhaps have a look.

First impression from a quick look, does starboard/win/shared/BUILD.gn need to include xb1_get_type.cc as a source or through a dependency? I'll take a deeper look on monday.

**Comment by iuriionishchenko** [comment]
Using ::starboard::shared::uwp::GetXboxType() causes build issue for win32 project. Should I add refences to uwp name space in the win32 project or it is batter to move ::starboard::shared::uwp::GetXboxType() to some common namespace?

**Comment by kaidokert** [comment]
> Using ::starboard::shared::uwp::GetXboxType() causes build issue for win32 project. Should I add refences to uwp name space in the win32 project or it is batter to move ::starboard::shared::uwp::GetXboxType() to some common namespace?

Shouldn't this condition actually go into `VideoDecoderUwp` class in `starboard/xb1/shared/video_decoder_uwp.h` ?

**Review by amurovanyi — APPROVED** [review]

**Review by TyHolc — APPROVED** [review]

**Comment by kaidokert** [comment]
I think this broke Xbox builds

**Pull request merged**